javascript/javascript3/week2/creating_database.sql

```diff
@@ -0,0 +1,39 @@
+
+/* Part 4: Creating a database */
+/* Using an entity relationship diagram, design the data model for an application of your choice. This can be anything, previous students have used a small business (with staff, offices, and job titles), a library (with books, genres, racks, members, and a borrowing log), or a farm (with animals, barns, and farmers). */
+/* Your application must include at least one many-to-many relationship and any supporting tables (linking tables) that are needed. The entity relationship diagram must describe what tables you will need, the columns in these tables, which column is the primary key, and the relationships between tables. */
+/* Next, using the entity relationship diagram as a starting point, write all the necessary CREATE TABLE statements to create all tables and relationships (foreign key constraints) for this data model. */
+/* Submit an image or PDF of your entity relationship diagram, and a .sql file with the CREATE TABLE statements. */
+
+
+CREATE DATABASE small_business;
+Use small_business;
+
+CREATE TABLE `staff` (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` varchar(255) NOT NULL,
+    `email` varchar(255) NOT NULL,
+    `phone` varchar(255) NOT NULL,
+    `hired_date` DATETIME NOT NULL
```

Comment:
That's true, I changed it to DATE, I thought they can be used the same way but the DATETIME can accept only date or date and time, that's not correct? :)

Reply:
I think if you have a DATETIME field and give it only a DATE it will fill up the TIME part with some default values. But it you only want dates, then you should use a DATE field, it uses less space and it's a way better practice.